CONTRIBUTING.md

```diff
@@ -1,75 +1,47 @@
-# Welcome to Reflex contributing guide! 🥳
+# Reflex Contributing Guidelines
 
-## Getting started
+For an extensive guide on the different ways to contribute to Reflex please visite our notion's [Contributing Guide](https://www.notion.so/reflex-dev/2107ab2bc166497db951b8d742748284?v=f0eaff78fa984b5ab15d204af58907d7).
 
-To navigate our codebase with confidence, see [Reflex Docs](https://reflex.dev/docs/getting-started/introduction) :confetti_ball:.
 
-### Discussions
 
-- Have a question? Want to discuss a feature? [Start a discussion](https://github.com/pynecone-io/pynecone/discussions)
-
-    We welcome any discussions and questions. We want to make sure that Reflex is the best it can be, and we can't do that without your help.
-
-### Issues
-
-* #### Create a new issue
-
-    If you spot a problem with anything in Reflex feel free to create an issue. Even if you are not sure if its a problem with the framework or your own code, create an issue and we will do our best to answer or resolve it.
-
-* #### Solve an issue
-
-    Scan through our [existing issues](https://github.com/pynecone-io/pynecone/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix. Any large issue changing the compiler of Reflex should brought to the Reflex maintainers for approval
+## Runing a Local Build of Reflex 
+Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
 
-Thank you for supporting Reflex!🎊
+**Pre-requisites:**
+- Python >= 3.7
+- Poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
```

Comment:
Technically you shouldn't need poetry to contribute. Should we mention that it's optional but that this guide will show the poetry way?